### PR TITLE
Feat/create-ride-offer

### PR DESCRIPTION
**Pull request by lupyana**

### CODE REVIEW

File: backend/yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,5571 @@
+Arguments:
```
Comment (makoscafee) [inline backend/yarn-error.log:1]: You should remember adding this file in .gitignore. as is not supposed to be pushed

File: backend/router/routes.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 const express = require('express');
+const bodyParser = require('body-parser');
 
 const router = express.Router(); // get an instance of the express Router
-
+// parse various different custom JSON types as JSON
+router.use(bodyParser.json());
```
Comment (makoscafee) [inline backend/router/routes.js]: Why is this used here? .I do not think this is necessary as body parser is already used in the application context

File: backend/router/routes.js [hunk 3]
```diff
@@ -36,11 +38,19 @@ const rides = [
...
+router.post('/', (req, res) => {
+  res.send('Got a POST request');
+});
 
 router.get('/rides', (req, res) => {
   res.status(200).json({ rides });
 });
 
+router.post('/rides', (req, res) => {
+  rides.push(req.body.newRide);
```
Comment (makoscafee) [inline backend/router/routes.js:47]: I am assuming you have to send a post request as in this format right?
``` js
{
 newRide { /*ride information here*/}
}
```

**Review by makoscafee — CHANGES_REQUESTED** [review]
All this looks good so far. I have left a few changes or rather areas that might need explanation.
General Comment:
- Add Fix #numberOfIssue in descriptions 
- Add yarn-error in .gitignore

File: backend/router/routes.js (hunk 3 above)
Comment (lupyana) [reply backend/router/routes.js:47]: Yes, that is correct

File: backend/router/routes.js (hunk 2 above)
Comment (lupyana) [reply backend/router/routes.js]: Fixed

File: backend/router/routes.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 const express = require('express');
 
 const router = express.Router(); // get an instance of the express Router
-
+// parse various different custom JSON types as JSON
```
Comment (makoscafee) [inline backend/router/routes.js]: nit: Remove this comment here does not make sense now

**Review by makoscafee — APPROVED** [review]

**Pull request merged**